ReactQt/runtime/src/qml/ReactWebView.qml

```diff
@@ -0,0 +1,22 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+
+import QtWebView 1.1
+import React 0.1 as React
+
+WebView {
+    id: webViewRoot
+    property var p_source: ''
+    property var webViewManager: null
+    property var flexbox: React.Flexbox {control: webViewRoot}
+    url: (p_source && p_source.uri) ? p_source.uri : ""
+
+    onWebViewManagerChanged: {
+        if (!webViewManager) {
+            return;
+        }
+        webViewManager.s_reload.connect(webViewRoot.reload);
```

Comment:
should `webViewManager` property be checked for null before used?

Reply:
@MaxRis ,  agree, it seems logical that initially it's null then when it's initialized onChange is invoked. But without this check I'm getting 
`Cannot read property 's_reload' of null`
Seems somehow onChange is invoked but the property is still null.

Reply:
@narek-h , I'm sorry, my comment is not relevant here, please, ignore